frontend/src/js/components/PageViewer/VirtualScroll.module.css

```diff
@@ -0,0 +1,24 @@
+.scrollArea {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Don't think these have any effect because you have the `.pages` element intervening between the `.scrollArea` and the `.page` elements.

If you put

```css
  display: flex;
  flex-direction: column;
  justify-content: center;
  ```
on `.pages`, it will center each `.page` element which is nice